articles/azure-functions/functions-scenario-database-table-cleanup.md

```diff
@@ -19,7 +19,9 @@ ms.author: glenga
 
 ---
 # Use Azure Functions to connect to an Azure SQL Database
-This topic shows you how to use Azure Functions to create a scheduled job that cleans up rows in a table in an Azure SQL Database. The new C# function is created based on a pre-defined timer trigger template in the Azure portal. To support this scenario, you must also set a database connection string as a setting in the function app. This scenario uses a bulk operation against the database. To have your function process individual CRUD operations in a Mobile Apps table, you should instead use [Mobile Apps bindings](functions-bindings-mobile-apps.md).
+This topic shows you how to use Azure Functions to create a scheduled job that cleans up rows in a table in an Azure SQL Database. The new C# function is created based on a pre-defined timer trigger template in the Azure portal. To support this scenario, you must also set a database connection string as an app setting in the function app. This scenario uses a bulk operation against the database. 
+
+To have your function process individual Create, Read, Update, and Delete (CRUD) operations in a Mobile Apps table, you should instead use [Mobile Apps bindings](functions-bindings-mobile-apps.md).
```

Comment:
Spell out what CRUD stands for

Reply:
@andresnava thanks for the close edit. Can you please change Create, Read, etc... to lowercase? Thanks!